packages/integration-tests/index.js

```diff
@@ -653,16 +731,26 @@ async function buildWorkflow(
   setProcessEnvironment(stackName, bucketName);
 
   const template = await getWorkflowTemplate(stackName, bucketName);
-  const { name, version } = collection || {};
-  const collectionInfo = collection
-    ? await new Collection().get({ name, version })
-    : {};
-  const providerInfo = provider
-    ? await new Provider().get({ id: provider.id })
-    : {};
-
-  template.meta.collection = collectionInfo;
-  template.meta.provider = providerInfo;
+
+  if (collection) {
```

Comment:
It looks like the previous code didn't handle this case either: but what if `collection` is an object with a `name` but not a `version`? Then the API request to get the collection will fail and `template.meta.collection` won't have the expected value. Should we check for both properties?

Reply:
A collection with a name but without a version is an invalid collection and would only be caused by a broken test. I'm fine leaving this the way it is and assuming that our integration tests are passing a valid collection config here.